app/views/content_items/how_government_works.html.erb

```diff
@@ -141,65 +141,37 @@
         <% if !@content_item.reshuffle_in_progress? %>
           <div class="govuk-grid-row feature-ministers">
             <div class="govuk-grid-column-one-half">
-              <p class="govuk-!-margin-bottom-1 govuk-heading-s">Prime Minister</p>
-
-              <a class="govuk-link gem-c-big-number__link" href="/government/ministers/prime-minister">
-                <%= render "govuk_publishing_components/components/big_number", {
-                  number: @content_item.ministerial_role_counts["prime_minister"],
-                } %>
-
-                <span class="govuk-visually-hidden">&nbsp;</span>
-                <span class="govuk-visually-hidden">
-                  Prime Minister
-                </span>
-              </a>
+              <%= render "govuk_publishing_components/components/big_number", {
```

Comment:
Oh that's nicer, all those `<span class="govuk-visually-hidden">&nbsp;</span` were a bit :grimacing:

Reply:
Yeah, I think it was so the label would display above the number but a screenreader would read the label after the number.  The big number component solves this by always putting the label after the number.